nav2_waypoint_follower/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
 project(nav2_waypoint_follower)
 
+# Default to C++17
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
This can be removed, C++17 is now standard in Humble/Rolling

Reply:
Done